src/base.iced

```diff
@@ -178,23 +178,38 @@ class Base
 
   #------
 
-  _v_include_pgp_hash : -> false
-  _v_require_full_pgp_hash : -> false
-  _v_pgp_km_to_hash : () -> @km()
-  _v_pgp_hash_dest : (body) -> body.key
+  _v_include_pgp_details : -> false
+  _v_require_pgp_details : -> false
+  _v_pgp_details_dest : (body) -> body.key
```

Comment:
This default only applies to `eldest`, right? Can we instead make the default an error (or `null` or something, which presumably causes an error later), and then make `eldest` specify its details dest explicitly?